### ISSUE TITLE
Lack of Texture Splatting support

### ISSUE THREAD

**Issue by eix128** [bjorn/tiled#2205]
Is there any implementation avalible for texture splatting ?

https://www.gamasutra.com/blogs/AndreyMishkinis/20130716/196339/Advanced_Terrain_Texture_Splatting.php

I need to make top down RTS game.
But tiled seems doesnt have any support for Texture splatting.

or does it have ?

**Comment by bjorn** [comment]
No, at the moment Tiled supports tile layers, object layers or image layers, but not layers with textures that are blended together based on alpha channels or more advanced fragment shader computations.

For doing such transitions on tile layers, commonly the transitions are pre-baked or hand-drawn. You can then tell Tiled which corner of your tiles corresponds to which type of terrain, so that it can automate the placement of the various transition tiles. See [the Terrain Brush documentation](http://doc.mapeditor.org/en/latest/manual/using-the-terrain-tool/) for how this works.

For supporting texture splatting, Tiled will need a new map rendering implementation that can take advantage of shaders, based on which such new functionality can then be developed. I'd welcome this type of stuff but it's quite a long way off. The "Tiled Quick" prototype (available on the `master` branch) is using OpenGL through the Qt Scene Graph and could in the future provide such functionality.
